soh/soh/Enhancements/randomizer/randomizer.h

```diff
@@ -19,6 +19,7 @@ class Randomizer {
     std::string ganonHintText;
     std::string ganonText;
     std::unordered_map<RandomizerSettingKey, u8> randoSettings;
+    std::unordered_map<RandomizerCheck, u16> scrubPrices;
```

Comment:
general naming thing, maybe `scrubItemPrices`?

Reply:
Renamed to randomizerMerchantPrices, will re-use for shopsanity